packages/common-all/src/fuse.ts

```diff
@@ -132,6 +95,18 @@ export type SerializedFuseIndex = ReturnType<
 
 type FuseEngineOpts = {
   mode?: DEngineMode;
+  /** If specified must be within 0-1 range. */
+  fuzzThreshold: number;
+};
+
+export const getThresholdValue = (configThreshold: number) => {
+  // Setting it to fallback threshold value in case configuration is incorrect.
+  let threshold = 0.2;
```

Comment:
instead of hardcoding, can we use `ConfigUtils.getDefaults()` so we get the fallback value from the config?

Reply:
Do we have a method like that already, doesnt come up for me when I do symbol search.

Reply:
i was going off memory. the method is `genDefaultConfig`